### PR TITLE
Website Portfolio Project: Local storage

### PR DESCRIPTION
**Pull request by JEFFKAY16**
## Website Portfolio Project: Local storage

In this update, we did the following:

- Added a function to set form inputs to local storage.
- Added a function to set input values from local storage.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -93,5 +96,21 @@ form.addEventListener('submit', (e) => {
...
+    localStorage.clear();
   }
 });
+
+function populateStorage() {
+  localStorage.setItem('name', document.querySelector('[type=text]').value);
+  localStorage.setItem('email', document.querySelector('[type=email]').value);
+  localStorage.setItem('textarea', document.querySelector('textarea').value);
+}
+formInput.forEach((input) => input.addEventListener('keyup', populateStorage));
```
Comment (ZeenatLawal) [inline index.js]: - [ ] Kindly refactor this function to meet this requirement: `You must create a single JavaScript object with all the data from the entire form and save it in local storage. Do not create one local storage entry per input field.` e.g.

```js
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```

File: index.js [hunk 2]
```diff
@@ -93,5 +96,21 @@ form.addEventListener('submit', (e) => {
...
+
+function setForm() {
+  document.querySelector('[type=text]').value = localStorage.getItem('name');
+  document.querySelector('[type=email]').value = localStorage.getItem('email');
+  document.querySelector('textarea').value = localStorage.getItem('textarea');
+}
+setForm();
```
Comment (ZeenatLawal) [inline index.js]: - [x] Kindly refactor this function to meet this requirement: `You must create a single JavaScript object with all the data from the entire form and save it in local storage. Do not create one local storage entry per input field.`

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @Wes-Isaac and @JEFFKAY16, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (JEFFKAY16) [reply index.js]: Hi @ZeenatLawal 

Thank you so much for the review, my partner and I have taken note of this. We are using a single in-buit javascript object that achieves the same thing as the project requirements. Creating an object won't store the data when the page is refreshed. @Wes-Isaac and I await your guidance please.

File: index.html [hunk 3]
```diff
@@ -252,6 +252,7 @@ <h1 class="form-header">I'm always interested in hearing about new projects, so
                         </label>
                     </div>
                     <div class="form-control">
+                        <textarea name="message" cols="30" maxlength="500" rows="10" required placeholder="Enter text"></textarea>
                         <textarea name="message" cols="30" maxlength="500" rows="10" placeholder="Enter text" required></textarea>
                         <img class="textarea-img" src="./images/Union.png" alt="union-icon">
```
Comment (ZeenatLawal) [inline index.html]: - [OPTIONAL] While making the changes required from the previous review, it seems a new input `textarea` was added. Kindly fix this before you merge the PR.

**Review by ZeenatLawal — APPROVED** [review]
Hi @Wes-Isaac and @JEFFKAY16, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] While making the changes required from the previous review, it seems a new input `textarea` was added. Kindly fix this before you merge the PR.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**